package.json

```diff
@@ -12,8 +12,8 @@
     "build": "tsoa spec-and-routes && swc ./src -d ./build --strip-leading-paths --copy-files",
     "start": "node build/index.js",
     "dev": "concurrently \"NODE_ENV=dev nodemon | pino-colada\" \"nodemon -x tsoa spec-and-routes --ignore src/routes.ts --ignore src/swagger.json\"",
-    "test:offchain": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/integration/offchain/*.test.ts",
-    "test:onchain": "NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/integration/onchain/*.test.ts",
+    "test:offchain": "PROXY_FOR=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/integration/offchain/*.test.ts",
+    "test:onchain": "PROXY_FOR=5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy NODE_ENV=test ./node_modules/.bin/mocha --config ./test/mocharc.json ./test/integration/onchain/*.test.ts",
```

Comment:
I'd suggest putting these in `test.env` and then providing the overide for `proxyless`. That way it's less to cleanup when we remove the option of not using a proxy